Dockerfile

```diff
@@ -101,6 +101,8 @@ USER ${USER}
 COPY --chown=user:user ci/makefiles/android.mk /tmp/android.mk
 RUN make --file /tmp/android.mk \
     && sudo rm /tmp/android.mk
+ENV ANDROIDSDK="${ANDROID_HOME}/android-sdk" \
+    ANDROIDNDK="${ANDROID_HOME}/android-ndk"
```

Comment:
This isn't documented, but I think `ANDROIDSDK` deprecated actually.
Maybe what you're looking for is `ANDROID_SDK_HOME` and `ANDROID_NDK_HOME`?
Also why not grouping higher up with the other environment variables?